lib/ziti-tunnel-cbs/ziti_tunnel_ctrl.c

```diff
@@ -57,6 +57,70 @@ const ziti_tunnel_ctrl* ziti_tunnel_init_cmd(uv_loop_t *loop, tunneler_context t
     return &CMD_CTX.ctrl;
 }
 
+
+int ziti_dump_to_log_cb(void* stringsBuilder, const char *fmt,  ...) {
+    static char line[4096];
+
+    va_list vargs;
+    va_start(vargs, fmt);
+    vsnprintf(line, sizeof(line), fmt, vargs);
+    va_end(vargs);
+
+    if (strlen(stringsBuilder) + strlen(line) > 8192) {
```

Comment:
8192 should be a macro or static const so it can be reused in multiple functions.